src/controllers/CheckYourAppealController.ts

```diff
@@ -1,25 +1,21 @@
-import { Session, SessionMiddleware } from 'ch-node-session-handler';
```

Comment:
Try and keep code looking like this as when its compressed its harder to read.  Unless @mouhajer-ch has another opinion?

Reply:
Will do. I think it was when the imports got reordered, it also formatted!

Reply:
I agree @CHdlewis, compressed makes the code harder to read. 
Something to remember, we could add [space-in-brackets](https://eslint.org/docs/rules/space-in-brackets) lint rule from eslint, when we migrate from the deprecated TSLint.